{"uid":"9cd54ecb431dd9f2","name":"View profile settings succesfully","fullName":"View profile settings succesfully","historyId":"View profile settings succesfully","time":{"start":1677727429761,"stop":1677727429768,"duration":7},"status":"broken","statusMessage":"An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:4444/session/868ef3810cb8c89d0e54880adc75a84e/screenshot. The exception message was: Connection refused (localhost:4444) -> Connection refused (localhost:4444) -> Connection refused","statusTrace":"OpenQA.Selenium.WebDriverException: An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:4444/session/868ef3810cb8c89d0e54880adc75a84e/screenshot. The exception message was: Connection refused (localhost:4444)\n ---> System.Net.Http.HttpRequestException: Connection refused (localhost:4444)\n ---> System.Net.Sockets.SocketException (111): Connection refused\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)\n   at System.Net.Sockets.Socket.<ConnectAsync>g__WaitForConnectWithCancellation|277_0(AwaitableSocketAsyncEventArgs saea, ValueTask connectTask, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.ConnectToTcpHostAsync(String host, Int32 port, HttpRequestMessage initialRequest, Boolean async, CancellationToken cancellationToken)\n   --- End of inner exception stack trace ---\n   at System.Net.Http.HttpConnectionPool.ConnectToTcpHostAsync(String host, Int32 port, HttpRequestMessage initialRequest, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.CreateHttp11ConnectionAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.AddHttp11ConnectionAsync(HttpRequestMessage request)\n   at System.Threading.Tasks.TaskCompletionSourceWithCancellation`1.WaitWithCancellationAsync(CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.GetHttp11ConnectionAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.SendWithVersionDetectionAndRetryAsync(HttpRequestMessage request, Boolean async, Boolean doRequestAuth, CancellationToken cancellationToken)\n   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpClient.<SendAsync>g__Core|83_0(HttpRequestMessage request, HttpCompletionOption completionOption, CancellationTokenSource cts, Boolean disposeCts, CancellationTokenSource pendingRequestsCts, CancellationToken originalCancellationToken)\n   at OpenQA.Selenium.Remote.HttpCommandExecutor.MakeHttpRequest(HttpRequestInfo requestInfo)\n   at OpenQA.Selenium.Remote.HttpCommandExecutor.Execute(Command commandToExecute)\n   --- End of inner exception stack trace ---\n   at OpenQA.Selenium.Remote.HttpCommandExecutor.Execute(Command commandToExecute)\n   at OpenQA.Selenium.WebDriver.Execute(String driverCommandToExecute, Dictionary`2 parameters)\n   at OpenQA.Selenium.WebDriver.GetScreenshot()\n   at SeleniumTest.Tests.Hooks.Hooks.CaptureScreenshot() in /home/sch/actions-runner/_work/GUI-Framework/GUI-Framework/3.Tests/UI/Hooks/Hooks.cs:line 176\n   at TechTalk.SpecFlow.Bindings.BindingInvoker.InvokeBinding(IBinding binding, IContextManager contextManager, Object[] arguments, ITestTracer testTracer, TimeSpan& duration)\n   at Allure.SpecFlowPlugin.AllureBindingInvoker.InvokeBinding(IBinding binding, IContextManager contextManager, Object[] arguments, ITestTracer testTracer, TimeSpan& duration)","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"BeforeFeature [10000]","time":{"start":1677727429750,"stop":1677727429760,"duration":10},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"CreateProject [1]","time":{"start":1677727429761,"stop":1677727429761,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"CreateItems [2]","time":{"start":1677727429761,"stop":1677727429761,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"BeforeScenario [10000]","time":{"start":1677727429761,"stop":1677727429761,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"testStage":{"status":"broken","statusMessage":"An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:4444/session/868ef3810cb8c89d0e54880adc75a84e/screenshot. The exception message was: Connection refused (localhost:4444) -> Connection refused (localhost:4444) -> Connection refused","statusTrace":"OpenQA.Selenium.WebDriverException: An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:4444/session/868ef3810cb8c89d0e54880adc75a84e/screenshot. The exception message was: Connection refused (localhost:4444)\n ---> System.Net.Http.HttpRequestException: Connection refused (localhost:4444)\n ---> System.Net.Sockets.SocketException (111): Connection refused\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)\n   at System.Net.Sockets.Socket.<ConnectAsync>g__WaitForConnectWithCancellation|277_0(AwaitableSocketAsyncEventArgs saea, ValueTask connectTask, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.ConnectToTcpHostAsync(String host, Int32 port, HttpRequestMessage initialRequest, Boolean async, CancellationToken cancellationToken)\n   --- End of inner exception stack trace ---\n   at System.Net.Http.HttpConnectionPool.ConnectToTcpHostAsync(String host, Int32 port, HttpRequestMessage initialRequest, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.CreateHttp11ConnectionAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.AddHttp11ConnectionAsync(HttpRequestMessage request)\n   at System.Threading.Tasks.TaskCompletionSourceWithCancellation`1.WaitWithCancellationAsync(CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.GetHttp11ConnectionAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.SendWithVersionDetectionAndRetryAsync(HttpRequestMessage request, Boolean async, Boolean doRequestAuth, CancellationToken cancellationToken)\n   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpClient.<SendAsync>g__Core|83_0(HttpRequestMessage request, HttpCompletionOption completionOption, CancellationTokenSource cts, Boolean disposeCts, CancellationTokenSource pendingRequestsCts, CancellationToken originalCancellationToken)\n   at OpenQA.Selenium.Remote.HttpCommandExecutor.MakeHttpRequest(HttpRequestInfo requestInfo)\n   at OpenQA.Selenium.Remote.HttpCommandExecutor.Execute(Command commandToExecute)\n   --- End of inner exception stack trace ---\n   at OpenQA.Selenium.Remote.HttpCommandExecutor.Execute(Command commandToExecute)\n   at OpenQA.Selenium.WebDriver.Execute(String driverCommandToExecute, Dictionary`2 parameters)\n   at OpenQA.Selenium.WebDriver.GetScreenshot()\n   at SeleniumTest.Tests.Hooks.Hooks.CaptureScreenshot() in /home/sch/actions-runner/_work/GUI-Framework/GUI-Framework/3.Tests/UI/Hooks/Hooks.cs:line 176\n   at TechTalk.SpecFlow.Bindings.BindingInvoker.InvokeBinding(IBinding binding, IContextManager contextManager, Object[] arguments, ITestTracer testTracer, TimeSpan& duration)\n   at Allure.SpecFlowPlugin.AllureBindingInvoker.InvokeBinding(IBinding binding, IContextManager contextManager, Object[] arguments, ITestTracer testTracer, TimeSpan& duration)","steps":[{"name":"Given  the user is logged in","time":{"start":1677727429761,"stop":1677727429763,"duration":2},"status":"failed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"attachments":[],"parameters":[],"stepsCount":1,"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[{"name":"CaptureScreenshot [10000]","time":{"start":1677727429768,"stop":1677727429769,"duration":1},"status":"broken","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"AfterFeature [10000]","time":{"start":1677727429770,"stop":1677727429771,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"labels":[{"name":"thread","value":"NonParallelWorker"},{"name":"host","value":"sch"},{"name":"feature","value":"Profile settings view"},{"name":"tag","value":"Smoke"},{"name":"tag","value":"Regression"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":8,"unknown":0,"total":9},"items":[{"uid":"eb75d22e622d2de1","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/732//#testresult/eb75d22e622d2de1","status":"passed","time":{"start":1677726600316,"stop":1677726607106,"duration":6790}},{"uid":"3e0dcf0040d428ec","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/730//#testresult/3e0dcf0040d428ec","status":"passed","time":{"start":1677724925500,"stop":1677724931897,"duration":6397}},{"uid":"5836381e6832f1","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/14//#testresult/5836381e6832f1","status":"passed","time":{"start":1677718783989,"stop":1677718790806,"duration":6817}},{"uid":"384a3db520ee16d","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/729//#testresult/384a3db520ee16d","status":"passed","time":{"start":1677714561280,"stop":1677714567811,"duration":6531}},{"uid":"2ce397d7e7221f11","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/728//#testresult/2ce397d7e7221f11","status":"passed","time":{"start":1677713188809,"stop":1677713195811,"duration":7002}},{"uid":"3d2d65bd1704d2db","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/727//#testresult/3d2d65bd1704d2db","status":"passed","time":{"start":1677710650804,"stop":1677710657143,"duration":6339}},{"uid":"299cbd0624985bbb","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/726//#testresult/299cbd0624985bbb","status":"passed","time":{"start":1677709868377,"stop":1677709874837,"duration":6460}},{"uid":"cbd9c42d095e7ff6","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/725//#testresult/cbd9c42d095e7ff6","status":"passed","time":{"start":1677709340505,"stop":1677709346858,"duration":6353}}]},"tags":["Regression","Smoke"]},"source":"9cd54ecb431dd9f2.json","parameterValues":[]}